<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>命令行快捷方式</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="w_copy">
        <property name="toolTip">
         <string>复制(Ctrl+Shift+C)</string>
        </property>
        <property name="text">
         <string>复制</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::EditCopy"/>
        </property>
        <property name="shortcut">
         <string>Ctrl+Shift+C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="w_cmd"/>
      </item>
      <item>
       <widget class="QToolButton" name="w_exec">
        <property name="toolTip">
         <string>执行(Ctrl+Enter)</string>
        </property>
        <property name="text">
         <string>执行</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
        </property>
        <property name="shortcut">
         <string>Ctrl+Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>工作目录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="w_workdir"/>
      </item>
      <item>
       <widget class="QToolButton" name="w_selectworkdir">
        <property name="toolTip">
         <string>选择工作目录</string>
        </property>
        <property name="text">
         <string>选择工作目录</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
      <item>
       <widget class="QListView" name="w_cmdlist">
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::DropAction::IgnoreAction</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="w_arglist">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="w_out"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>w_copy</tabstop>
  <tabstop>w_cmd</tabstop>
  <tabstop>w_exec</tabstop>
  <tabstop>w_workdir</tabstop>
  <tabstop>w_selectworkdir</tabstop>
  <tabstop>w_cmdlist</tabstop>
  <tabstop>w_arglist</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>on_w_selectworkdir_clicked()</slot>
 </slots>
</ui>
